sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint @typescript-eslint/member-ordering: 0 */
```

Comment:
Is it possible to put this under the copyright header and still work?

Reply:
I'll try!

Reply:
It does work!